### PR TITLE
Varvara/feature/parse  test data

### PR DESCRIPTION
**Pull request by varvarashcherbyna7**

### CODE REVIEW

File: app/commands/job.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+                is_deleted=job.is_deleted,
+            )
+
+            db_service: m.Service = session.scalar(sa.select(m.Service).where(m.Service.id == service_id))
+            assert db_service, f"Service with id [{db_service}] not found"
+
+            session.add(new_job)
+            session.flush()
+
+            new_job
```
Comment (sa1am8) [inline app/commands/job.py]: what purpose of it?

File: app/commands/job.py [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+            )
+
+            session.add(new_job_service)
+
+            new_job_service
+
+            if with_print:
+                log(log.DEBUG, "Job with title [%s] created", job.title)
+
+        jobs
```
Comment (sa1am8) [inline app/commands/job.py]: and this one?

File: api/utility/job.py [hunk 3]
```diff
@@ -8,8 +8,9 @@
 
 from app import models as m
 from app.logger import log
+from app import schema as s
 
-from .address import create_addresses
+# from .address import create_addresses
```
Comment (StoliarskyiKostiantyn) [inline api/utility/job.py:13]: needed?

File: api/utility/job.py [hunk 4]
```diff
@@ -29,25 +30,25 @@ def create_job(db: Session, user_id: int, is_pending=False) -> m.Job:
 
     if not addresses:
         log(log.ERROR, "No addresses found")
-        create_addresses(db)
+        # create_addresses(db)
```
Comment (StoliarskyiKostiantyn) [inline api/utility/job.py:33]: same

File: app/commands/addresses.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+from pathlib import Path
+
+import sqlalchemy as sa
+
+from app import models as m
+from app import schema as s
+from app.database import db
+
+MODULE_PATH = Path(__file__).parent
+JSON_FILE = MODULE_PATH / ".." / ".." / "data" / "address.json"
```
Comment (StoliarskyiKostiantyn) [inline app/commands/addresses.py:11]: move to .env?

File: app/commands/user.py [hunk 6]
```diff
@@ -8,8 +9,227 @@
...
+            else:
+                val[0] = phone
+        else:
+            val[0] = val[0].strip().replace("(", "").replace(")", "").replace("-", "")
+            if len(val[0]) == 10:
+                val[0] = f"38{val[0]}"
+            elif len(val[0]) == 9:
+                val[0] = f"380{val[0]}"
+
+    for i, v in enumerate(values):
```
Comment (sa1am8) [inline app/commands/user.py:76]: do we need this?

File: migrations/versions/12bb37c7d575_job_description_len.py [hunk 7]
```diff
@@ -0,0 +1,38 @@
+"""job_description_len
+
+Revision ID: 12bb37c7d575
+Revises: 5e1c08ca7d4a
+Create Date: 2024-02-26 17:07:21.461612
+
+"""
+from alembic import op
+import sqlalchemy as sa
```
Comment (sa1am8) [inline migrations/versions/12bb37c7d575_job_description_len.py:9]: 5e1c08ca7d4a_job_location_id_none and 12bb37c7d575_job_description_len contains similiar logic of changing job.description length

**Review by sa1am8 — APPROVED** [review]

File: app/commands/job.py [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+from app import schema as s
+from app.database import db
+from app.logger import log
+from config import config
+
+from .utility import SEARCH_IDS, authorized_user_in_google_spreadsheets
+
+CFG = config()
+
+# ['ID', 'owner_id', 'worker_id', 'title', 'description', 'service', 'location', 'address', 'created_at', 'started_at', 'finished_at', 'rate_client', 'rate_worker']
```
Comment (StoliarskyiKostiantyn) [inline app/commands/job.py:16]: needed?

File: config.py [hunk 9]
```diff
@@ -56,6 +56,11 @@ class BaseConfig(BaseSettings):
 
     MAX_USER_SEARCH_RESULTS: int = 10
 
+    # for test data from google spreadsheets
+
+    SCOPES: list[str] = ["https://www.googleapis.com/auth/spreadsheets"]
```
Comment (StoliarskyiKostiantyn) [inline config.py:61]: maybe it must be in .env ????

**Review by StoliarskyiKostiantyn — APPROVED** [review]

File: migrations/versions/12bb37c7d575_job_description_len.py (hunk 7 above)
Comment (varvarashcherbyna7) [reply migrations/versions/12bb37c7d575_job_description_len.py:9]: thanks for the review)

File: config.py (hunk 9 above)
Comment (varvarashcherbyna7) [reply config.py:61]: thanks for the review!

**Pull request merged**